app/views/admin/people/edit.html.erb

```diff
@@ -1,12 +1,16 @@
-<% page_title "Edit " + @person.name %>
+<% content_for :page_title, "Edit #{@person.name}" %>
+<% content_for :title, "Edit #{@person.name}" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @person)) %>
 
-<div class="row">
-  <section class="col-md-8">
-    <h1>Edit person</h1>
-    <p class="warning">Warning: changes to people appear instantly on the live site.</p>
-    <%= render partial: "form", locals: {person: @person, show_instantly_live_warning: true} %>
-  </section>
-  <div class="col-md-4">
-    <%= legacy_simple_formatting_sidebar %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/warning_text", {
+      text: "Warning: changes to people appear instantly on the live site.",
```

Comment:
Carlos advised, remove 'Warning:' from message

Reply:
Sorry yes this was discussed in a call earlier. He advised remove 'Warning:' from the string's as a warning is explanitory

Reply:
Have actioned. Thanks!